### PR TITLE
Modify rule S4275: Update code samples

### PR DESCRIPTION
**Pull request by francoismora**
Fixes SonarSource/SonarJS#3759

### CODE REVIEW

File: rules/S4275/javascript/rule.adoc [hunk 1]
```diff
@@ -5,41 +5,66 @@ include::../description.adoc[]
...
 
-  public setX(val: number) {
-    this.x = val;
+  getY() {
+    return this.#y;
   }
 
-  public getY() {
-    return this.y;
+  set y(val: number) {
```
Comment (yassin-kammoun-sonarsource) [inline rules/S4275/javascript/rule.adoc:66]: I am a bit confused here. The non compliant snippet for this setter defines `setY` while here we use the syntax `set y`. Is that on purpose?

File: rules/S4275/javascript/rule.adoc [hunk 2]
```diff
@@ -5,41 +5,66 @@ include::../description.adoc[]
...
 }
+
+let x = 0;
+let y = 0;
+const obj2 = {};
+Object.defineProperty(obj2, 'x', {
+  get() {
+    return y;
+  }
+});
```
Comment (yassin-kammoun-sonarsource) [inline rules/S4275/javascript/rule.adoc:43]: Can we add `// Noncompliant` annotations to highlight the issue?

Comment (yassin-kammoun-sonarsource) [reply rules/S4275/javascript/rule.adoc:43]: We are missing the compliant counterpart of these.

**Review by yassin-kammoun-sonarsource — CHANGES_REQUESTED** [review]

File: rules/S4275/javascript/rule.adoc (hunk 1 above)
Comment (francoismora) [reply rules/S4275/javascript/rule.adoc]: I guess I pasted the wrong code snippet. Fixed.

File: rules/S4275/javascript/rule.adoc (hunk 2 above)
Comment (francoismora) [reply rules/S4275/javascript/rule.adoc]: Fixed.

**Review by yassin-kammoun-sonarsource — APPROVED** [review]

**Pull request merged**